src/objectTranslations/Contact-ja.objectTranslation

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <webLinks>
+        <label>関係する個人ビューア</label>
```

Comment:
I just cloned the repo and downloaded this branch, and it appears fine (with Japanese) on my system.  So I think we can assume this is just some weird gitHub display issue.